core/src/cats/postgresql.cc

```diff
@@ -584,6 +584,7 @@ bool BareosDbPostgresql::SqlQueryWithoutHandler(const char* query, int)
 
       num_fields_ = (int)PQnfields(result_);
       Dmsg1(500, "we have %d fields\n", num_fields_);
+      fields_fetched_ = false;
```

Comment:
This is probably fine for our code but this should always happen in this function, even if it did not work.  Since we override `result_` (and free it if necessary) in all cases, the field names will always point to nothing or to freed memory anyways.

Reply:
applied changes.